pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -152,6 +155,7 @@ func newNode(name string, label map[string]string) *v1.Node {
 			Name:      name,
 			Labels:    label,
 			Namespace: metav1.NamespaceNone,
+			Tenant:    metav1.NamespaceNone,
```

Comment:
guess you mean TenantNone, instead of NamespaceNone, though they are both empty strings.

Reply:
this Tenant is removed